src/test/java/zipkin/sparkstreaming/adjuster/zipkin/BinaryAnnotationTrimAdjusterPropertiesTest.java

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
we don't put license headers on example code. it distracts from the content, which isn't being distributed etc.